x/auth/types/stdtx.go

```diff
@@ -12,21 +12,23 @@ import (
 )
 
 var (
-	_ sdk.Tx = (*StdTx)(nil)
+	_ sdk.Tx        = (*StdTx)(nil)
+	_ sdk.Signature = (*StdSignature)(nil)
+	_ sdk.Signature = StdSignature{}
 
 	maxGasWanted = uint64((1 << 63) - 1)
 )
 
 // StdTx is a standard way to wrap a Msg with Fee and Signatures.
 // NOTE: the first signature is the fee payer (Signatures must not be nil).
 type StdTx struct {
-	Msgs       []sdk.Msg      `json:"msg"`
-	Fee        StdFee         `json:"fee"`
-	Signatures []StdSignature `json:"signatures"`
-	Memo       string         `json:"memo"`
+	Msgs       []sdk.Msg       `json:"msg"`
+	Fee        StdFee          `json:"fee"`
+	Signatures []sdk.Signature `json:"signatures"`
```

Comment:
Because of this, this PR might is probably breaking and we should mark it as such. I'm pretty sure the amino bytes will be different here.